iOS/MSMapsModules/MSMapsModules/AppDelegate.h

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Can you put the test app in a separate project so it's easier to make a module out of the geojson code?